### PR TITLE
aws_transit_gateway_multicast_group_member

### PR DESCRIPTION
**Pull request by soumyo13**
Signed-off-by: Soumyodeep Karmakar <soumyo.k13@gmail.com>

### Description

It contains all the singular and plural resources of the above resource. No integration tests with terraform are done as there is no terraform resource available.

### Issues Resolved
This is a new resource. There are no issues to be fixed.

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [x] New functionality includes integration tests/controls
- [x] New Terraform resources
- [x] Documentation provided or updated for resources 
- [ ] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: docs/resources/aws_transit_gateway_multicast_group_members.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+`transit_gateway_multicast_domain_id` _(required)_
+
+The transit_gateway_multicast_domain_id is the ID of the multicast domain transit gateway.
+
+For additional information, see the [AWS documentation on AWS EC2 TransitGatewayMulticastGroupMember](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-transitgatewaymulticastgroupmember.html).
+
+## Properties
+
+| Property | Description|
```
Comment (sathish-progress) [inline docs/resources/aws_transit_gateway_multicast_group_members.md:30]: please add fields if possible

Comment (soumyo13) [reply docs/resources/aws_transit_gateway_multicast_group_members.md]: Done

**Review by sathish-progress — APPROVED** [review]
LGTM

**Comment by sathish-progress** [comment]
please update readme with resource links

also the fields for plural resource are unquoted ..

**Pull request merged**